plugin/main/src/kotlinx/benchmark/gradle/BenchmarksPlugin.kt

```diff
@@ -92,6 +104,12 @@ constructor() : Plugin<Project> {
         }
     }
 
+    private fun createBenchmarksExtension(project: Project): BenchmarksExtension {
+        return project.extensions.create(BENCHMARK_EXTENSION_NAME, BenchmarksExtension::class.java, project).apply {
+            kotlinCompilerVersion.convention(DEFAULT_KOTLIN_COMPILER_VERSION)
```

Comment:
Why do we need a default version if it will be overridden by the Kotlin version used in a user project?

Reply:
kotlinx-benchmarks can be used in pure Java projects, where KGP isn't present.

https://github.com/Kotlin/kotlinx-benchmark/blob/v0.4.11/examples/java/build.gradle